starknet_py/tests/e2e/contract_interaction/interaction_test.py

```diff
@@ -165,24 +165,6 @@ async def test_call_uninitialized_contract(gateway_account_client):
     assert "Requested contract address 0x1 is not deployed." in err.value.message
 
 
-@pytest.mark.asyncio
-async def test_deploy_throws_on_no_compilation_source(account_client):
-    with pytest.raises(ValueError) as exinfo:
-        await Contract.deploy(client=account_client)
-
-    assert "One of compiled_contract or compilation_source is required." in str(
-        exinfo.value
-    )
-
-
-@pytest.mark.asyncio
-async def test_wait_for_tx(account_client, map_compiled_contract):
-    deployment = await Contract.deploy(
-        compiled_contract=map_compiled_contract, client=account_client
-    )
-    await account_client.wait_for_tx(deployment.hash)
```

Comment:
I think we should still have this test but perhaps with different kind of transaction.